server/index.ts

```diff
@@ -47,6 +47,23 @@ const useIPFSPinnig = process.env.USE_IPFS === 'true';
 const ignoreVoteEndConstraint: boolean =
   process.env.IGNORE_VOTE_END_CONSTRAINT === 'true';
 
+/**
+ * We need to ensure there's at most 1 mongo connection for the lifespan of this service.
+ * Creating this connection here allows the connection logic to be isolated to this file.
+ */
+const mongoConnectionURI = process.env.MONGODB_TRIBUTE_DAOS_URI;
+console.log('Connecting to mongo...')
+export const snapshotHubMongoDb = new MongoClient(
```

Comment:
I forget if Mongo logs an error internally, or if we need to handle it in `.catch(error)`. e.g. Set a bad connection string to test.